src/java/org/apache/cassandra/io/sstable/SSTableZeroCopyWriter.java

```diff
@@ -233,4 +236,20 @@ private void write(AsyncStreamingInputPlus in, long size, SequentialWriter write
             throw new FSWriteError(e, writer.getPath());
         }
     }
+
+    private static class ZeroCopySequentialWriter extends SequentialWriter
+    {
+        private ZeroCopySequentialWriter(File file, SequentialWriterOption option, boolean strictFlushing)
+        {
+            super(file, ByteBufferUtil.EMPTY_BYTE_BUFFER, option, strictFlushing);
+        }
+
+        @Override
+        public void write(byte[] b, int off, int len) throws IOException
```

Comment:
I can't actually find a single code path that hits this outside of unit tests... trying to see what production configs or conditions hit this, but as far as i can tell we hard code the async channel in netty, so this is a test only code path?